system/Commands/Generators/Views/model.tpl.php

```diff
@@ -6,22 +6,22 @@
 
 class {class} extends Model
 {
-    protected $DBGroup              = '{DBGroup}';
-    protected $table                = '{table}';
-    protected $primaryKey           = 'id';
-    protected $useAutoIncrement     = true;
-    protected $insertID             = 0;
-    protected $returnType           = '{return}';
-    protected $useSoftDeletes       = false;
-    protected $protectFields        = true;
-    protected $allowedFields        = [];
+    protected $DBGroup          = '{dbGroup}';
```

Comment:
This could theoretically be a breaking change. Maybe we should add a note somewhere about template files that they are "internal"?

Reply:
I have a doc comment under `app/Config/Generators.php` although on reading it is only implicit that the templates are internal. Can that be enough or should I add there a more explicit note?

Reply:
No that sounds good! Sorry I missed it.